web/tests/EducationBenchmarking.Web.A11yTests/TestSupport/Config.cs

```diff
@@ -0,0 +1,17 @@
+namespace EducationBenchmarking.Web.A11yTests.TestSupport;
+using Microsoft.Extensions.Configuration;
+
+public class Config
+{
+    public static IConfiguration Instance => new ConfigurationBuilder()
+#if !DEBUG
+    .AddJsonFile("appsettings.json", optional: false)
+#else
+        .AddJsonFile("appsettings.local.json", optional: false)
+#endif
+        .Build();
+
+    public static string LandingPage => Instance.GetValue<string>("Urls:LandingPage")!;
+    public static string ChooseSchoolPage => Instance.GetValue<string>("Urls:ChooseSchoolPage")!;
+    public static string BenchmarkingPage => Instance.GetValue<string>("Urls:BenchmarkingPage")!;
```

Comment:
The page is CompareYourSchool - it avoid confusion later down the line we should make sure we are naming things that align with the service.

Reply:
updated in latest commits.